prac_05/emails.py

```diff
@@ -0,0 +1,31 @@
+"""
+CP1404 | Prac_05 | Dannielle Jones
+Get email, extract name and store in dictionary.
+"""
+
+
+def main():
+    """Create dictionary of emails-to-names."""
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        name = get_name_from_email(email)
+        correct = input("Is your name {}? (Y/n) ".format(name))
+        if correct.upper() != "Y" and correct != "":
+            name = input("Name: ")
+        email_to_name[email] = name
+        email = input("Email: ")
+
+    for email, name in email_to_name.items():
+        print("{} ({})".format(name, email))
```

Comment:
Great variable names, logic is good. Docstrings are nice and to the point. Wish I could offer more constructive feedback but this is good work.